user_story/model/user_story.py

```diff
@@ -378,13 +378,13 @@ def do_disapproval(self, cr, uid, ids, context=None):
     def get_body_approval(self, cr, uid, i, context=None):
         usname = self.browse(cr, uid, i).name
         username = self.pool.get('res.users').browse(cr, uid, uid).name
-        link = '#id={i}&view_type=form&model=user.story'.format(i=i)
+        link = '#id=%(i)s&view_type=form&model=user.story' % dict(i=i)
         return _(u'''<html><div>
-                 <h2>{usname}</h2>
-                 <p>The user {user} has approved the user Story
-                 <a href="{link}">See what we are talking about here</a>
-                 </div></html>'''.format(usname=usname, user=username,
-                                         link=link))
+                 <h2>%(usname)s</h2>
+                 <p>The user %(user)s has approved the user Story
+                 <a href="%(link)s">See what we are talking about here</a>
+                 </div></html>''' % (dict(usname=usname,
```

Comment:
The focus is search the bad cases
How to search bad cases? `% (dict(`
How to fix them? `% dict(`